<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeletedNotes</class>
 <widget class="QMainWindow" name="DeletedNotes">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>683</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>683</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>960</width>
    <height>683</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>82</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(27, 43, 66);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_36">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="DNHomeButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PNG/House4.png</normaloff>../PNG/House4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="DNUserButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PNG/profile4.png</normaloff>../PNG/profile4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QScrollArea" name="DNScrollArea">
          <property name="styleSheet">
           <string notr="true">background: white;
border-style:solid; 
border-width: 0.5px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>896</width>
             <height>586</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
